src/requests/cache/sorted_set_put_element.rs

```diff
@@ -0,0 +1,105 @@
+use momento_protos::cache_client::{SortedSetElement, SortedSetPutRequest};
+
+use crate::requests::cache::MomentoRequest;
+use crate::simple_cache_client::prep_request_with_timeout;
+use crate::{CacheClient, CollectionTtl, IntoBytes, MomentoResult};
+
+/// Request to add an element to a sorted set. If the element already exists, its score is updated.
+/// Creates the sorted set if it does not exist.
+///
+/// # Arguments
+///
+/// * `cache_name` - The name of the cache containing the sorted set.
+/// * `sorted_set_name` - The name of the sorted set ot add an element to.
+/// * `value` - The value of the element to add. Must be able to be converted to a Vec<u8>.
+/// * `score` - The score of the element to add.
+///
+/// # Optional Arguments
+///
+/// * `collection_ttl` - The time-to-live for the collection. If not provided, the client's default time-to-live is used.
+///
+/// # Examples
+/// Assumes that a CacheClient named `cache_client` has been created and is available.
+/// ```
+/// # fn main() -> anyhow::Result<()> {
+/// # use momento_test_util::create_doctest_client;
+/// # tokio_test::block_on(async {
+/// use momento::CollectionTtl;
+/// use momento::requests::cache::sorted_set_put_element::SortedSetPutElement;
+/// use momento::requests::cache::sorted_set_put_element::SortedSetPutElementRequest;
+/// # let (cache_client, cache_name) = create_doctest_client();
+/// let sorted_set_name = "sorted_set";
+///
+/// let put_element_request = SortedSetPutElementRequest::new(
+///     cache_name.to_string(),
+///     sorted_set_name.to_string(),
+///     "value",
+///     1.0
+/// ).with_ttl(CollectionTtl::default());
+///
+/// let create_cache_response = cache_client.send_request(put_element_request).await?;
+///
+/// assert_eq!(create_cache_response, SortedSetPutElement {});
+/// # Ok(())
+/// # })
+/// # }
+pub struct SortedSetPutElementRequest<S: IntoBytes, E: IntoBytes> {
```

Comment:
nit I think the convention for naming these generic types is to name them the first letter of whatever field they are going into, so we did `E` for "elements", and this one would be `V` I think.

Reply:
Good find. That's a copy/paste error from when I copied it from set_add_elements.